spec/lib/health_check/json_search_client_spec.rb

```diff
@@ -11,9 +11,7 @@ def search_response_body
           "link" => "/b"
         }
       ],
-      "suggested_queries" => %w(
-A
-B)
+      "suggested_queries" => %w(A B)
```

Comment:
This looks like it will change the nature of the test slightly, because on the left it has newline characters in and on the right it has spaces.

It doesn't seem like it matters to any of these tests though, and I think this class is actually completely useless. We should be able to replace it with gds-api-adapters.

Reply:
Have checked on irb and it looks like it makes the same object :)